samples/nrf9160/modem_shell/src/gnss/gnss_interface.c

```diff
@@ -576,7 +576,7 @@ static void get_agps_data(struct k_work *item)
 	struct nrf_cloud_rest_context rest_ctx = {
 		.connect_socket = -1,
 		.keep_alive = false,
-		.timeout_ms = CONFIG_NRF_CLOUD_REST_RECV_TIMEOUT * MSEC_PER_SEC,
+		.timeout_ms = NRF_CLOUD_REST_TIMEOUT_NONE,
```

Comment:
Similar comment to other places where these timeout values were changes as "NONE".
Eventhough "timeout_ms" is not currently used in nrf cloud side and not working in rest client/http client, I think it would be still better to retain actual timeout value in these and it will be someday actually working.

Reply:
the timeout is disabled in the nrf_cloud_rest library, so if/when the http_client is fixed, the nrf_cloud_rest library will also need to be updated.  these values could be updated at that time.
also, do we want to be using the socket receive kconfig value for a different purpose?

Reply:
OK. That's sounds like a plan. Sckt rcv timeout is different but yeah, little bit overlapping with http_client response timeout.